roles/edpm_ovs/molecule/common/update/prepare.yml

```diff
@@ -0,0 +1,80 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Prepare
+  hosts: all
+  vars:
+    scenario_name: "{{molecule_yml['scenario']['name']}}"
+  pre_tasks:
+    - name: Importing test_vars
+      ansible.builtin.include_vars:
+        file: ./test_vars.yml
+  tasks:
+    - name: "Include edpm_users with non default config for ovs"
+      ansible.builtin.include_role:
+        name: "osp.edpm.edpm_users"
+      vars:
+        edpm_users_users:
+          - {"name": "hugetlbfs", "gid": "42477", "group_only": true}
+        edpm_users_extra_dirs: []
+
+    - name: Select openvswitch test version to install
+      ansible.builtin.set_fact:
+        # Unversioned name will install the latest available version
+        ovs_test_version:
+          "{{'rdo-openvswitch' if scenario_name == 'update_no_op'
```

Comment:
can we define prepare.yml so that it imports this yaml with the `ovs_test_version` set to the "package version" to install instead? References to test scenario names are ugly and fragile.

Reply:
Also, I wonder if rdo- prefix in the name of the package is problematic. It probably means that when we decide to run the scenarios against RHOSO repos, this will fail. Not sure it's something we plan to do though.

Reply:
> can we define prepare.yml so that it imports this yaml with the `ovs_test_version` set to the "package version" to install instead? References to test scenario names are ugly and fragile.

Sure I can break this out into a separate yaml and just import it here. But that will still make a decision based on the `scenario_name`. Or I am not understanding you correctly? 

> Also, I wonder if rdo- prefix in the name of the package is problematic. It probably means that when we decide to run the scenarios against RHOSO repos, this will fail. Not sure it's something we plan to do though.

Yes this crossed my mind, but I see the whole edpm_ansible branch is written based on RDO repositories and hardcoded "antelope" "wallaby" etc. references  Updating this will be trivial but first we will need to decide how the whole edpm_ansible will run different branches. I hope I am explaining myself correctly.

Reply:
Agree with Ihar here, Can't we just pass edpm_ovs_packages with required package name in test scenario itself instead of calculating here?

Reply:
I moved the version to install into each scenario molecule.yml. Under [`provisioner`][`env`]. Thats the best place I could find for it. Let me know